src/concurrency/ocf_pio_concurrency.c

```diff
@@ -136,9 +136,26 @@ static int ocf_pio_lock_slow(struct ocf_alock *alock,
 	return ret;
 }
 
+static uint32_t ocf_pio_lock_get_entries_count(struct ocf_alock *alock,
+		struct ocf_request *req)
+{
+	uint32_t i, count = 0;
+	ocf_cache_line_t entry;
+
+	for (i = 0; i < req->core_line_count; i++) {
+		entry = ocf_pio_lock_get_entry(alock, req, i);
+		if (entry == OUT_OF_RANGE)
```

Comment:
I guess the majority of requests will be modifying entries that are in range. What do you think about adding `unlikely()`?